examples/dbscan/ArborX_DBSCANVerification.hpp

```diff
@@ -115,7 +115,8 @@ bool verifyBoundaryAndNoisePoints(ExecutionSpace const &exec_space,
           if (!is_boundary && labels(i) != -1)
           {
 #ifndef __SYCL_DEVICE_ONLY__
-            printf("Noise point does have index -1: %d [%d]\n", i, labels(i));
+            printf("Noise point does not have index -1: %d [%d]\n", i,
+                   labels(i));
```

Comment:
The change in formatting is slightly unexpected, but whatever...

Reply:
🤷

Reply:
> How is it unexpected?

@dalg24 It means that a real noise point has index different from -1, which means it was clustered, which should not have happened.